Desktop/results/ploteditoraxismodel.cpp

```diff
@@ -20,7 +20,9 @@ void AxisModel::setAxisData(const Json::Value & axis)
 
 	Json::Value	settings = axis.get(	"settings",		Json::objectValue);
 
-	setTitle(		tq(						settings.get(	"title",		""	).asString()));
+	Json::Value title = settings.get("title", "");
+
+	setTitle(		tq(						title.isString() ? title.asString() : ""));
```

Comment:
```suggestion
	setTitle(		tq(						title.isString() ? title.asString() : title.toStyledString()));
```

Reply:
Maybe nicer?

Reply:
I don't think: `toStyledString` adds curly brackets `{}`

Reply:
Yes I get that but then at least you see something...

The alternative is nothing, which drops information...

Reply:
I don't understand: the title must be an empty string.

Reply:
Like, the fix is to avoid titles that are not a string to not crash jasp right?
So the title could be a number?
In case it is a number, doing 'toStyledString' might be slightly better than just dropping it?
Although I guess then it is even better to do:
```suggestion
	setTitle(		tq(						title.isString() ? title.asString() : title.isDouble() ? ColumnUtils::doubleToString(title.asDouble(), false) : title.isInt() ? std::to_string(title.asInt()) : ""));
```
No?